src/components/2019/year-in-review/News.js

```diff
@@ -0,0 +1,534 @@
+import React from 'react'
+import s from 'styled-components'
+
+import { MoreStories, PhotoGallery, Date, SectionMainText, PrimaryStory, StyledStoryLink, SecondaryStory, PrimaryStoryLine, SubSectionHeader, MainHeader } from './Typograph'
+
+const Wrapper = s.div`
+    /* Util classes */
+    .right-border {
+        border-right: 1px solid rgba(255, 255, 255, 0.3);
+    }
+
+    .right-border-inverse {
+        border-right: 1px solid rgba(0, 0, 0, 0.3);
+    }
+
+    .narrow-col-right {
+        padding-right: 24px;
+    }
+
+    .narrow-col-left {
+        padding-left: 24px;
+    }
+`
+
+const Section = s.div`
+    background-color: ${({ inverse }) => inverse ? "white" : "#131313"};
+    padding: ${({ inverse }) => inverse ? "80px 0px" : "120px 0px"};
+
+`
+const News = () => {
+  return (
+    <Wrapper id="news">
+        <Section id="news">
+            <div class="container">
+                <SubSectionHeader>PENN ADMISSIONS SCANDAL</SubSectionHeader>
+                <div class="row">
+                    <div class="col-md-6 right-border narrow-col-right">
+                        <img class="img-fluid" src="https://s3.amazonaws.com/snwceomedia/dpn/f880ab9e-bc8a-4f3d-a44b-541a31e6c13c.sized-1000x1000.jpg" />
+                        <MainHeader>College admissions scandal hits Penn men’s basketball</MainHeader>
+                        <SectionMainText>Former Penn men’s basketball coach Jerome Allen has remained a key witness for the investigation into the college admissions bribery scandal that came to light in March 2019. He plead guilty in October 2018 to accepting bribes in exchange for recruitment of a player. In March, Allen testified in a federal trial that Philip Esformes paid him to facilitate his son’s recruitment in 2015. Although the national scandal that broke days after Allen's testimony did not name Penn, Dean of Admissions Eric Furda said Penn would consider revising its recruitment process. During Esformes' trial, news broke that he had also sent more than $400,000 to the consultant at the center of the national scandal after contacting him about his son’s recruitment. Esformes’ trial ended in April when he was found guilty on 20 charges, including bribery. In July, a judge sentenced Allen to four years probation, community service, and a fine, avoiding jail time. The next day, Penn Athletics removed Allen from its Hall of Fame.</SectionMainText>
+                    </div>
+                    <div class="col-md-6 narrow-col-left">
+                        <div class="row">
+                            <div class="col-md-6">
+                                <a href="https://www.thedp.com/article/2019/03/upenn-admissions-bribery-eric-furda-jerome-allen-college"><PrimaryStory> Furda says Penn will revisit admissions procedures in light of national bribery scandal </PrimaryStory></a>
```

Comment:
when nesting React components I prefer using multiple lines instead of having really long lines like this.